class-gwiz-gf-code-chest.php

```diff
@@ -213,6 +213,7 @@ public function init() {
 
 		add_filter( 'gform_register_init_scripts', array( $this, 'register_init_script' ), 99, 1 );
 		add_filter( 'gform_register_init_scripts', array( $this, 'maybe_register_custom_js_scripts_first' ), 100, 1 );
+		add_filter( 'gform_register_init_scripts', array( $this, 'maybe_execute_custom_php' ));
```

Comment:
I think this should be `add_action` since `gform_register_init_scripts` is an action, right?

```suggestion
		add_action( 'gform_register_init_scripts', array( $this, 'maybe_execute_custom_php' ));
```

Reply:
Also, I think [plugins_loaded](https://developer.wordpress.org/reference/hooks/plugins_loaded/) might be a better option. Because snippets should run in all Gravity Forms context. E.g. when forms are rendered, when submitted, viewing entries, form editor etc.

Reply:
Perfect. I switched to 'add_action' and used the 'plugins-loaded' hooks instead.